fastlane/Fastfile

```diff
@@ -124,7 +125,7 @@ platform :ios do
   desc 'Creates a new release branch from the current trunk'
   lane :complete_code_freeze do |options|
     ios_completecodefreeze_prechecks(options)
-    ios_localize_project
+    update_localization
```

Comment:
For easy reference, here's what that lane does: https://github.com/woocommerce/woocommerce-ios/blob/a0b20cf1fc90f0ced98bab42f20b5ca116d95b65/fastlane/Fastfile#L809-L812

`update_localization` is boarder line redundant, given it only calls an action, i.e. `fastlane update_localization` is equivalent to `fastlane run ios_localize_project`. 

But, I think it's handy to have it because you can have tab completion in the terminal for lanes, but not for running actions. That is, if we have a dedicated lane it's easier to run and discover in the terminal than a dedicated action.

Aside from that, we'd would have wanted to extract the localization call anyway once moving to the new setup, which require more SLOCs